core/src/test/resources/logback-test.xml

```diff
@@ -1,19 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-
-    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-        <target>System.out</target>
-        <encoder>
-            <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1} [%X{akkaSource}] - %m%n%xException</pattern>
-        </encoder>
-    </appender>
-
-    <logger name="org.apache.cassandra" level="ERROR" />
-    <logger name="com.datastax" level="INFO" />
-    <logger name="io.netty" level="ERROR" />
-
-    <root level="INFO">
-        <appender-ref ref="CONSOLE"/>
-    </root>
-
+  <appender name="CONSOLE"
+  class="ch.qos.logback.core.ConsoleAppender">
+    <target>System.out</target>
+    <encoder>
+      <pattern>%date{MM/dd HH:mm:ss} %-5level[%thread] %logger{1}
+      [%X{akkaSource}] - %m%n%xException</pattern>
```

Comment:
does that result in line breaks in the log messages?

Reply:
Not sure but will revert, some auto . formatting some where